### DISCUSSION TITLE
Can I import Classic Shell settings .xml into Open Shell?

### DISCUSSION THREAD

**Discussion by Wizlon** [Open-Shell/Open-Shell-Menu#1906] [Q&A] [answered]
I currently have Classic Shell on my Win 10 desktop, however I have a new laptop arriving tomorrow which will be dedicated to Win 11.  Will my Classic Shell settings .xml file be readable by Open Shell?

**Comment by ge0rdi** [comment, marked as answer]
Yes.
`Open-Shell` is compatible with `Classic Shell`'s settings.

**Comment by Wizlon** [comment]
Will it matter that I am going from W10 to W11?

**Reply by ge0rdi** [reply]
No.

**Reply by Wizlon** [reply]
Thanks, wasn't sure.
